source/shaders/glyphquad.frag

```diff
@@ -0,0 +1,39 @@
+precision mediump float;
+
+@import ./facade.frag;
+
+#if __VERSION__ == 100
+    #define fragColor gl_FragColor
+    #extension GL_OES_standard_derivatives : enable
+#else
+    layout(location = 0) out vec4 fragColor;
+#endif
+
+
+uniform sampler2D u_glyphs;
+
+varying vec2 v_texture_coord;
+
+void main(void)
+{
+    /* requires blend: glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA); */
+
+    float d = texture(u_glyphs, v_texture_coord).r;
+
+    /**
+     * using if-statement and discard can slow down performance:
+     * it's bad for IMR, TBR, TBDR and early-Z optimization
+     * https://stackoverflow.com/questions/8509051/is-discard-bad-for-program-performance-in-opengl
+     * But it is necessary: overlapping glyphs (like in 'ft') should not fight each other
+     */
+    if(d < 0.45)
```

Comment:
from a performance perspective: this is irrelevant, leave it or remove it, will probably change nothing.
BUT: for the interactive text selection feature where we need an additional label ID buffer to resolve the position and label from to, e.g., set the Cursor to the correct position/glyph

Reply:
Ok. In the early days, there were z-fighting problem with overlapping glyphs (e.g., 'ft'), but I can't reproduce them right now. So I remove the discard for later Cursor Interaction.